### PR TITLE
feat(designer-ui): Update file picker to use Fluent UI v9 components

### LINKED ISSUE #4916: Convert file picker to Fluent UI 9
### Describe the Bug with repro steps

Existing file picker was ported from old designer and does not match Fluent (esp. v9) design patterns.

### What type of Logic App Is this happening in?

Standard (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Workflow JSON

_No response_

### Screenshots or Videos

image

### Browser

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ek68794998**
- **Please check if the PR fulfills these requirements**

* [x] The commit message follows our guidelines
* [x] Tests for the changes have been added (for bug fixes/features)
* [ ] Docs have been added / updated (for bug fixes / features)

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Feature

- **What is the current behavior?** (You can also link to an open issue here)

Existing file picker uses old design language items are sorted arbitrarily.

- **What is the new behavior (if this is a feature change)?**

File picker uses Fluent v9 components, modern design patterns, and sorts items properly. Closes #4916.

- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

It shouldn't. I renamed one file that is re-exported (`filepickereditor.tsx` &rarr; `filepickerEditor.tsx`) but its exported members were not changed.

- **Please Include Screenshots or Videos of the intended change**:

fluent-filepicker

### CODE REVIEW

File: libs/designer-ui/src/lib/picker/filepickerPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { FilePickerPopoverHeader } from './filepickerPopoverHeader';
```
Comment (ek68794998) [inline libs/designer-ui/src/lib/picker/filepickerPopover.tsx:1]: Looks like there's an a11y issue where you can't tab from header to the file rows.

File: libs/designer-ui/src/lib/picker/filepickerPopoverHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+          index={index}
+          item={segment}
+          isLast={index === currentPathSegments.length - 1}
+        />
+      ))}
+      {overflowItems ? (
+        <>
+          <Menu>
+            <MenuTrigger disableButtonEnhancement={true}>
+              <Tooltip content={'More' /* TODO */} relationship="label" withArrow={true}>
```
Comment (ek68794998) [inline libs/designer-ui/src/lib/picker/filepickerPopoverHeader.tsx:89]: Missed a couple TODO i18n here.

Comment (ek68794998) [reply libs/designer-ui/src/lib/picker/filepickerPopoverHeader.tsx]: Fixed.

File: libs/designer-ui/src/lib/picker/filepickerPopover.tsx (hunk 1 above)
Comment (ek68794998) [reply libs/designer-ui/src/lib/picker/filepickerPopover.tsx:1]: Fixed.

**Review by hartra344 — APPROVED** [review]

**Pull request merged**